### PR TITLE
Complete jQuery pager example

### PR DESCRIPTION
**Pull request by MikeVitik**

### CODE REVIEW

File: libs/jquery/index.html [hunk 1]
```diff
@@ -8,6 +8,47 @@
 </head>
 
 <body>
+    <style>
+        :host {
```
Comment (williamvinogradov) [inline libs/jquery/index.html:12]: Minor comment.

As far as I know, the host selector is for the shadow root
[Link to the MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:host_function)

So, we can remove it here.

File: libs/jquery/index.html [hunk 2]
```diff
@@ -112,6 +180,14 @@
             document.querySelector('#button').addEventListener('click', () => {
                 seelctBox.option('readOnly', !seelctBox.option('readOnly'))
             })
+            const pager = new window.DevExpress.ui.dxPager(document.querySelector('#pager'), {});
+            document.querySelector('.page-sizes').addEventListener('change', ({ srcElement }) => {
```
Comment (williamvinogradov) [inline libs/jquery/index.html:184]: Minor comment.

For spike check it's okay, but in a future version, we must remove event listeners on component destroy :)

File: libs/react/src/components/pager/views/dxPagerPageNumberView.tsx [hunk 3]
```diff
@@ -8,18 +8,24 @@ interface DxPagerPageNumberViewProps {
...
 
-const DxPagerPageNumberView = ({data: {viewModel, selectPage}}: DxPagerPageNumberViewProps) => {
+const DxPagerPageNumberView = ({ data: { viewModel, selectPage } }: DxPagerPageNumberViewProps) => {
   return (
     <div className="dx-pager-pages">
       {
-        viewModel.items.map((item) =>
-          item.template({data: {item, selectPage}})
+        viewModel.items.map((item, key) => {
+          const ItemComponent = item.template as any;
```
Comment (williamvinogradov) [inline libs/react/src/components/pager/views/dxPagerPageNumberView.tsx:21]: Minor comment.

I understand this is a health check of the possible template solution for jQuery.
But, I can't leave this without a comment :)

**Review by williamvinogradov — DISMISSED** [review]

File: libs/jquery/index.html (hunk 2 above)
Comment (williamvinogradov) [reply libs/jquery/index.html:180]: Here is my mistake, this place doesn't need removeEventListener :)

**Review by williamvinogradov — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**